docs/en-us/web/custom-module.md

```diff
@@ -1,40 +1,40 @@
-# 自定义 Web 模块
+# Custom Modules
 
-使用 Hippy 开发过程中，当前的能力可能无法满足一些特定场景，这时候就需要使用第三方库或者通过自定义实现来提供额外的能力，下面就来介绍如何提供一个自定义模块
+In the development of using Hippy, it is possible that the builtin module won't meet the need of certain circumstances. In this case, you can augment the module by encapsulation or including some third-party functionalities inside the module.
 
-# 模块的扩展
+# Modules' Extensions
 
-扩展模块主要包括：
+Modules' extensions can be achieved by:
 
-1. 创建 `HippyWebModule` 的子类
-2. 设置 `Moduel` 的 `name` 属性
-3. 实现 `Module` 需要暴露给前端的 `API`
+1. Extend from `HippyWebModule` class
+2. Set the `name` property of the custom `Module`
+3. Implement the `API` that exposed to Front-end for custom `Module` 
```

Comment:
Implement the `API` that exposed to Front-end for custom `Module`

->  
Implement the module `API` that need to be exposed to Front-end